pandas_datareader/data.py

```diff
@@ -56,6 +66,42 @@ def get_quote_google(*args, **kwargs):
     return GoogleQuotesReader(*args, **kwargs).read()
 
 
+def get_tops_iex(*args, **kwargs):
+    return IEXTops(*args, **kwargs).read()
```

Comment:
I would prob do these inside each function this is odd to import everything (though that's the existing style)

e.g.

``
def get_tops_iex(....):
    from pandas_datareader.iex.tops import TopsReader
    return TopsReader(....)
```
further would be nice to have a docstring here